--- a/download_and_build_arrow.cmake
+++ b/download_and_build_arrow.cmake
@@ -62,9 +62,9 @@
     endif()

     if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        set(ARROW_CMAKE_PRESET ninja-debug-minimal)
+        set(ARROW_BUILD_TYPE Debug)
     else()
-        set(ARROW_CMAKE_PRESET ninja-release-minimal)
+        set(ARROW_BUILD_TYPE RelWithDebInfo)
     endif()

     if(CMAKE_VERSION VERSION_GREATER_EQUAL "4.0")
@@ -101,7 +101,12 @@
         LOG_BUILD ON
         LOG_INSTALL ON
         CMAKE_ARGS
-        --preset ${ARROW_CMAKE_PRESET}
+        -DCMAKE_BUILD_TYPE=${ARROW_BUILD_TYPE}
+        -DARROW_BUILD_MINIMAL=ON
+        -DCMAKE_C_COMPILER_LAUNCHER=
+        -DCMAKE_CXX_COMPILER_LAUNCHER=
+        -DCMAKE_C_FLAGS=
+        -DCMAKE_CXX_FLAGS=
         -DARROW_BOOST_USE_SHARED=OFF
         -DARROW_BUILD_SHARED=${ARROW_BUILD_SHARED}
         -DARROW_BUILD_STATIC=${ARROW_BUILD_STATIC}
